packages/react-core/src/demos/Animations/examples/Animations.tsx

```diff
@@ -0,0 +1,1699 @@
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  <Flex flexWrap={{ default: 'nowrap' }}>
+                    <FlexItem>
+                      <Icon status="success">
+                        <CheckCircleIcon />
+                      </Icon>
+                    </FlexItem>
+                    <FlexItem>
+                      <span>Successful assignment</span>
+                    </FlexItem>
+                  </Flex>
+                </DescriptionListTerm>
+                <DescriptionListDescription>
+                  Successfully assigned default/example to ip-10-0-130-149.ec2.internal
+                </DescriptionListDescription>
+                <DescriptionListDescription>
+                  <Timestamp date={new Date('2023-06-17T11:13')} dateFormat="medium" timeFormat="short" />
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  <Flex flexWrap={{ default: 'nowrap' }}>
+                    <FlexItem>
+                      <Spinner size="md" aria-label="loading spinner" />
+                    </FlexItem>
+                    <FlexItem>
+                      <span>Pulling image</span>
+                    </FlexItem>
+                  </Flex>
+                </DescriptionListTerm>
+                <DescriptionListDescription>Pulling image "openshift/hello-openshift"</DescriptionListDescription>
+                <DescriptionListDescription>
+                  <Timestamp date={new Date('2023-06-17T10:59')} dateFormat="medium" timeFormat="short" />
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+              <DescriptionListGroup>
+                <DescriptionListTerm>
+                  <Flex flexWrap={{ default: 'nowrap' }}>
+                    <FlexItem>
+                      <Icon status="success">
+                        <CheckCircleIcon />
+                      </Icon>
+                    </FlexItem>
+                    <FlexItem>
+                      <span>Created container</span>
+                    </FlexItem>
+                  </Flex>
+                </DescriptionListTerm>
+                <DescriptionListDescription>Created container hello-openshift</DescriptionListDescription>
+                <DescriptionListDescription>
+                  <Timestamp date={new Date('2023-06-17T10:45')} dateFormat="medium" timeFormat="short" />
+                </DescriptionListDescription>
+              </DescriptionListGroup>
+            </DescriptionList>
+          </CardBody>
+          <Divider />
+          <CardFooter>
+            <a href="#">View all events</a>
+          </CardFooter>
+        </Card>
+      </Fragment>
+    );
+  };
+
+  const CardStatus: FunctionComponent = () => {
```

Comment:
Having these components being defined from within another component is potentially problematic, and breaks the rules of hooks for the hooks you're using within them.